### ISSUE TITLE
Custom shapes for the card

### ISSUE THREAD

**Issue by tonyshkurenko** [Meetic/Shuffle#16]
In my project I have to implement circle shapes for cards, but cards have shadows and squared layout, how to ignore that?
Thanks for the answer

**Comment by Joadar** [comment]
Hi, have you tried to override the xml files to remove the shadows and add you own layout?

**Comment by antonshkurenko** [comment]
Yeah, we did something like this. Downloaded sources and override some resources
